ee/clickhouse/queries/trends/total_volume.py

```diff
@@ -61,23 +50,72 @@ def _total_volume_query(self, entity: Entity, filter: Filter, team_id: int) -> T
                     **content_sql_params,
                     parsed_date_to=trend_event_query.parsed_date_to,
                     parsed_date_from=trend_event_query.parsed_date_from,
-                    **trend_event_query.active_user_params
+                    **trend_event_query.active_user_params,
                 )
             else:
                 content_sql = VOLUME_SQL.format(event_query=event_query, **content_sql_params)
 
             null_sql = NULL_SQL.format(trunc_func=trunc_func, interval_func=interval_func)
             params["interval"] = filter.interval
             final_query = AGGREGATE_SQL.format(null_sql=null_sql, content_sql=content_sql)
-            return final_query, params, self._parse_total_volume_result(filter)
+            return final_query, params, self._parse_total_volume_result(filter, entity, team_id)
 
-    def _parse_total_volume_result(self, filter: Filter) -> Callable:
+    def _parse_total_volume_result(self, filter: Filter, entity: Entity, team_id: int) -> Callable:
         def _parse(result: List) -> List:
             parsed_results = []
             for _, stats in enumerate(result):
                 parsed_result = parse_response(stats, filter)
+                parsed_result.update(
+                    {"persons_urls": self._get_persons_url(filter, entity, team_id, parsed_result["days"])}
+                )
                 parsed_results.append(parsed_result)
 
+            parsed_result.update({"filter": filter.to_dict()})
             return parsed_results
 
         return _parse
+
+    def _parse_aggregate_volume_result(self, filter: Filter, entity: Entity, team_id: int) -> Callable:
+        def _parse(result: List) -> List:
+            _, seconds_in_interval, _ = get_time_diff(
+                filter.interval, filter.date_from, filter.date_to, team_id=team_id
+            )
+            time_range = enumerate_time_range(filter, seconds_in_interval)
+            filter_params = filter.to_dict()
+            extra_params = {
+                "entity_id": entity.id,
+                "entity_type": entity.type,
+            }
+            parsed_params = deep_dump_object({**filter_params, **extra_params})
+            return [
+                {
+                    "aggregated_value": result[0][0] if result and len(result) else 0,
+                    "days": time_range,
```

Comment:
Is there some way we can get this from the query response rather than enumerating outside. It would be good to remove any change of misalignment

Reply:
e.g. it should be theoretically possible to return all the information we need in the query results, and only rely on this.

Reply:
aggregated values don't have time series values. The days are still required to be passed back to the frontend because the underlying linegraph component that's used for pie charts and tables still do some manipulations on the days list. Once that's decoupled, we shouldn't even have to return the tdays field here